inc/html.class.php

```diff
@@ -3923,14 +3931,19 @@ static function printPager($start, $numrows, $target, $parameters, $item_type_ou
    /**
     * Display the list_limit combo choice
     *
-    * @param $action    page would be posted when change the value (URL + param) (default '')
-    * @param $display   display the pager form if true, return it if false
+    * @param $action             page would be posted when change the value (URL + param) (default '')
```

Comment:
Variable type missing beetween param and variable name

Reply:
Fixed in commit f34462212c26558b8c52df65266ef7db60458c7e.

Reply:
Fixed.